pom.xml

```diff
@@ -2079,7 +2079,6 @@
                 <checkstyle.skip>true</checkstyle.skip>
                 <enforcer.skip>true</enforcer.skip>
                 <license.skip>true</license.skip>
-                <maven.test.skip>true</maven.test.skip>
```

Comment:
yeah, better to leave this out. For even quicker builds when just server needs to be rebuilt we can specify this ourselves.